File: MyProfile/Models/Education.cs
using System;
using System.Data.Common;

namespace MyProfile.Models
{
    public class Education
    {
        public int Id {get; set;}
        public string School { get; set;}
        public string Area { get; set;}
        public string StudyType { get; set;}
        public DateTime StartDate { get; set;}
        public DateTime EndDate { get; set;}
    }
}

File: MyProfile/client/src/components/education/Education.js
import { Card, CardBody } from "reactstrap"
import './education.css'

export const Education = ({education}) => {
    return (
        <>
        <div className="education">
        <Card
        color="secondary"
        >
            <CardBody>
                <h1>{education.school}</h1>
                <p>Study: {education.area}</p>
                <p>Type: {education.studyType}</p>
                <footer>Start Date: {education.startDate}</footer>
                <footer>End Date: {education.endDate}</footer>
            </CardBody>
        </Card>
        <br/>
        </div>
        </>
    )
}